internal/language/language.go

```diff
@@ -97,6 +98,10 @@ var lookup = [...]languageData{
 		"python2", "Python 2", ".py", true, "python", "2.7.14",
 		Executable{constants.ActivePython2Executable, false},
 	},
+	{
+		"ruby", "Ruby", ".rb", true, "ruby", "3.0.1",
+		Executable{constants.RubyExecutable, false},
```

Comment:
It might be worth checking if the default languages info endpoint in the mediator returns this info as well - this data is currently mostly a fallback.

Reply:
That's only used when initializing a project. In my tests it wasn't an issue, and this story only covers activate/install.